sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/ListBlobsIncludeItem.java

```diff
@@ -31,13 +31,13 @@
     TAGS("tags"),
 
     /** Enum value immutabilitypolicy. */
-    IMMUTABILITY_POLICY("immutabilitypolicy"),
+    IMMUTABILITYPOLICY("immutabilitypolicy"),
 
     /** Enum value legalhold. */
-    LEGAL_HOLD("legalhold"),
+    LEGALHOLD("legalhold"),
```

Comment:
@alzimmermsft I suspect this will get flagged as a breaking change. Any thoughts on why the generator changed this formatting and if I should change something with the generator or add a configuration?

Reply:
Not exactly sure what cause this change, @weidongxu-microsoft do you know off-hand? Otherwise I can take a look into finding the cause

Reply:
@alzimmermsft AFAIK, there is no recent change on how enum name is deduced.

From the swagger, it is "legalhold". I don't think codegen will be so clear to add the underscore between 2 words without any hints.
https://github.com/Azure/azure-rest-api-specs/blob/storage-dataplane-preview/specification/storage/data-plane/Microsoft.BlobStorage/preview/2020-12-06/blob.json#L11665-L11692

And I didn't see any customization about this.

Maybe this is previously done by hand?

Reply:
One possible enhancement on swagger is the `values` under `x-ms-enum`, which enable specifying a `name` for each enum value.
https://azure.github.io/autorest/extensions/#x-ms-enum

Let me know if codegen works with this (I am not sure but I think likely works).

Reply:
Oh this looks like it was [changed recently](https://github.com/Azure/azure-sdk-for-java/commit/7550193a0485a76f1acc6ec19370cc1363d73363#diff-f348b6bf2023f3d866d24fc0090b3c8099243c5f6bbeb9a5b75be2a028389b6d) to include the underscore? Still looking into how this passed CI

Reply:
I've determined that this was changed manually in response to some apiview feedback before it was released. The editor didn't know it was a generated file. I am going to manually edit it for now to get the apiview out. [An issue](https://github.com/Azure/autorest.java/issues/1207) has been filed on auto rest to support this sort of change via customizations.